### PR TITLE
Provider display

### PR DESCRIPTION
**Pull request by vinyll**
## Banner with a phone number

screen shot 2017-05-29 at 11 25 55 pm

## Banner with a phone number and an email

screen shot 2017-05-29 at 11 26 05 pm

## Details

screen shot 2017-05-29 at 11 26 15 pm

> Note: I'm not totally happy about how the banner moves to the left when we open the details, but I'm not sure how to deal with it.

### CODE REVIEW

**Comment by vinyll** [comment]
And we should remove the phone number and the email from the details

**Comment by fuentesloic** [comment]
ooooh wahou ... you are PO, DEV & ... designer :)

That look really good and better !

File: _includes/profile.html [hunk 1]
```diff
@@ -1,73 +1,21 @@
 <footer>
-  {% assign profile = site.data.profile[page.profile1] %}
+  {% for profile_name in page.profiles %}
+  {% assign profile = site.data.profile[profile_name] %}
   <article style="display:{{ profile.display; }}">
```
Comment (fuentesloic) [inline _includes/profile.html]: It's perfect with an array !

**Comment by fuentesloic** [comment]
I will just fix some banner default like: 
capture d ecran 2017-05-29 a 18 44 21
and font-size & flex-direction for: 
capture d ecran 2017-05-29 a 18 45 41
I will do that tomorrow morning

**Comment by fuentesloic** [comment]
capture d ecran 2017-05-30 a 10 48 58
capture d ecran 2017-05-30 a 10 49 05

**Comment by vinyll** [comment]
The phone number has disapeared from the display.
I'm sure it may work from my phone, but how can I view the number when browsing from my laptop or my tablet? How can I write the number on a piece of paper for later?

can we just display the number and eventually make it clickable?
The icons are probably a good idea! :)

**Comment by vinyll** [comment]
Same for the email f course…

**Comment by fuentesloic** [comment]
Two purpose: 
- I put in details the number & the email displayed
OR
- I open a popup like details with the number or the email (according wich icon you had selected)

**Comment by fuentesloic** [comment]
capture d ecran 2017-05-30 a 11 51 09

**Comment by vinyll** [comment]
Much better :)
Je sais pas si "détails" parle, mais testons…

**Comment by fuentesloic** [comment]
@vinyll: "description" sounds like better ?

**Comment by fuentesloic** [comment]
# full desktop:
capture d ecran 2017-05-31 a 10 53 03
# medium desktop
capture d ecran 2017-05-31 a 10 53 20
# small desktop:
capture d ecran 2017-05-31 a 10 53 27
# tablet:
capture d ecran 2017-05-31 a 10 53 34
# phone with detail close:
capture d ecran 2017-05-31 a 10 53 56
# phone with detail open:
capture d ecran 2017-05-31 a 10 53 47

**Comment by vinyll** [comment]
good for me. Approved.
Plese try to squash your micro fix commits into the one where it was supposed to go.

**Comment by fuentesloic** [comment]
I don't squash all of them into just one ?

**Pull request merged**